src/looprestoration_tmpl_16.rs

```diff
@@ -1047,6 +938,7 @@ unsafe extern "C" fn loop_restoration_dsp_init_x86(
     c: *mut Dav1dLoopRestorationDSPContext,
     bpc: libc::c_int,
 ) {
+    use crate::src::looprestoration::*;
```

Comment:
I'd prefer if we avoided these `::*` imports, but if they're temporary until we genericize all of `looprestoration`, it's fine in that case, but could you add a `TODO` comment saying it's temporary like the `pub`s?

Reply:
Yeah the imports are just temporary until the init fns and the `extern` decls are in the same module again. I've added comments accordingly.